samples/NativeCatalog/App.js

```diff
@@ -85,17 +100,23 @@ class ManualSigningScreen extends Component<{}> {
             flexDirection: 'row',
             height: 40,
             alignItems: 'center',
-            padding: 10,
+            padding: -10,
```

Comment:
This looked bad on iOS. On Android it looks okay with `-10`.

Reply:
I think it would actually look nicer with this (at least on iOS, since the view is auto sized and the text doesn't go outside the safe area. I haven't tested it on Android though.)

```
<SafeAreaView
  style={{
    flexDirection: 'row',
    alignItems: 'center',
  }}>
```

Do you think we could use that?

Reply:
Thanks @steviki! In https://github.com/PSPDFKit/react-native/pull/339/commits/1223bab7f903c13eb7bc8417c4e6fea06382361d I switched to `SafeAreaView` and it looks great on iOS.

@irgendeinich when you have a minute can you please test the first two examples on Android to see `SafeAreaView` works correctly?

Reply:
Using `SafeAreaView` also works great on Android.

| Manual Signing Example | Watermark Example |
| --- | --- |
| <img width="372" alt="Screen Shot 2020-03-18 at 7 29 11 AM" src="https://user-images.githubusercontent.com/7443038/76956464-7b63de80-68ea-11ea-9dfe-951ca54161c1.png"> | <img width="372" alt="Screen Shot 2020-03-18 at 7 29 30 AM" src="https://user-images.githubusercontent.com/7443038/76956472-7f8ffc00-68ea-11ea-9e14-7f83596cdcec.png"> |